src/network/connection.cpp

```diff
@@ -197,13 +207,7 @@ bool Connection::sendShortString(const char* str) {
 	return true;
 }
 
-bool Connection::sendPacketType(uint8_t type) {
-	MUST_TRANSFER_BOOL(sendByte(0));
-	MUST_TRANSFER_BOOL(sendByte(0));
-	MUST_TRANSFER_BOOL(sendByte(0));
-
-	return sendByte(type);
-}
+bool Connection::sendPacketType(uint8_t type) { return sendInt(type); }
```

Comment:
Not sure if that is what is intended. is PacketType 3Byte Reserve and then 1Byte of Data, or is it 4Byte 1 Number?

And i think no 1 line functions. We prefer the 3 line if?

Reply:
This is not an if, also, this is just what the formatter does.